fang/mysql_migrations/migrations/2023-08-17-102017_create_fang_tasks/up.sql

```diff
@@ -3,17 +3,23 @@
 
 -- docker exec -ti mysql mysql -u root -pfang -P 3360 fang -e "$(catn fang/mysql_migrations/migrations/2023-08-17-102017_create_fang_tasks/up.sql)"
 
+ /*
+        why `metadata` and `error_message` are not a TEXT ?
+        MySQL TEXT type, I think it is stored as a BLOB.
+        So that breaks FromRow trait, implemented in lib.rs line 183
+    */
+
 CREATE TABLE fang_tasks (
     id VARCHAR(36) DEFAULT (uuid()) PRIMARY KEY,
-    metadata JSON NOT NULL,
-    error_message TEXT,
+    metadata VARCHAR(2048) NOT NULL,
```

Comment:
maybe it can be just `VARCHAR`?

Reply:
![image](https://github.com/ayrat555/fang/assets/48651252/e01fbbf9-1e79-4d47-a673-0c85c2ba5056)

There is not `VARCHAR` type in MySQL (yeap we know this is kinda trash)

Reply:
it seems mysql supports json type?

https://dev.mysql.com/doc/refman/5.7/en/json.html

Reply:
yes, i have to change this for mysql type and for sqlite type.

also the way this field is encoded and decoded in the DB

Reply:
[fix serde_json::Value decoding and encoding](https://github.com/ayrat555/fang/pull/141/commits/aafb352b71b946f7f6d1be6292460873f5b12730)

this should address your concern